src/schema/types/signupMutation.graphql

```diff
@@ -0,0 +1,10 @@
+input SignupInput {
```

Comment:
I'm seeing that you've defined here the `input` that I suggested you above. Maybe you can create a new file called `user-input.graphql`, move this new type there and use it in both places. What do you think?